src/layout.ts

```diff
@@ -9,6 +9,7 @@ import { IIterator, ArrayIterator } from '@lumino/algorithm';
 import * as Blockly from 'blockly';
 
 import { BlocklyManager } from './manager';
+import { define_jupyter_theme } from './utils';
```

Comment:
Now we can import the theme directly, instead of a function that registers a new theme each time.

```suggestion
import { THEME } from './utils';
```